### PR TITLE
Added Inactivity Penalty tests 

### PR DESCRIPTION
**Pull request by doctoc**
Description : 
These tests were used for checking the bug raised in Issue #10. The bug is invalid now as inactivity penalty is being applied as expected.

Changes :
GRACE_PERIOD constant added in constants.js for testing purposes.
Defined as the max allowed inactive epochs for which there is no inactivity penalty.

Added Inactivity Penalty tests in StakeManager.js
1. Staker should be penalised when no. of inactive epochs > GRACE_PERIOD
2. Staker should not be penalised when no. of inactive epochs <= GRACE_PERIOD
3. Staker should be penalised even if they restake in GRACE_PERIOD and commit/reveal outside GRACE_PERIOD because restaking is not considered an activity.

### CODE REVIEW

**Comment by abhishekvispute** [comment]
Hey @doctoc 
As we merged https://github.com/razor-network/contracts/pull/99 
There are some conflicts 
Please rebase and resolve them

**Review by abhishekvispute — DISMISSED** [review]
LGTM
@SkandaBhat Please do Squash and Merge

**Review by dev1644 — CHANGES_REQUESTED** [review]
Hi @doctoc , also run `npm run lint:fix`, then I think we are good tgo go.

File: test/StakeManager.js [hunk 1]
```diff
@@ -167,5 +168,124 @@ describe('StakeManager', function () {
       staker = await stakeManager.getStaker(2);
       assert(String(staker.stake) === String(stake), 'Stake should not change');
     });
+
+    it('should penalize staker if number of inactive epochs > grace_period' , async function(){
```
[the remark this answers is not in the document]
Comment (SkandaBhat) [reply test/StakeManager.js:180]: +1

[the remark this answers is not in the document]
Comment (doctoc) [reply test/StakeManager.js]: Done . Please review

**Review by dev1644 — APPROVED** [review]

**Review by abhishekvispute — APPROVED** [review]

**Pull request merged**